### PR TITLE
Run rustfmt on muxers

### PR DESCRIPTION
**Pull request by tomaka**
Runs `rustfmt`. Please say a word if you think you have ongoing work that conflicts with this PR.

### CODE REVIEW

File: muxers/mplex/src/codec.rs [hunk 1]
```diff
@@ -176,27 +208,33 @@ impl Encoder for Codec {
...
+                endpoint: Endpoint::Dialer,
+            } => ((substream_id as u64) << 3 | 4, Bytes::new()),
+            Elem::Reset {
+                substream_id,
+                endpoint: Endpoint::Listener,
+            } => ((substream_id as u64) << 3 | 5, Bytes::new()),
+            Elem::Reset {
+                substream_id,
+                endpoint: Endpoint::Dialer,
+            } => ((substream_id as u64) << 3 | 6, Bytes::new()),
```
Comment (twittner) [inline muxers/mplex/src/codec.rs]: This is a terrible format.

File: muxers/mplex/src/codec.rs [hunk 2]
```diff
@@ -207,7 +245,10 @@ impl Encoder for Codec {
         let data_len_bytes = encode::usize(data_len, &mut data_buf);
 
         if data_len > MAX_FRAME_SIZE {
-            return Err(IoError::new(IoErrorKind::InvalidData, "data size exceed maximum"));
+            return Err(IoError::new(
+                IoErrorKind::InvalidData,
+                "data size exceed maximum",
+            ));
```
Comment (twittner) [inline muxers/mplex/src/codec.rs]: Why?

File: muxers/mplex/src/codec.rs [hunk 3]
```diff
@@ -227,15 +268,23 @@ mod tests {
         let mut enc = Codec::new();
         let endpoint = Endpoint::Dialer;
         let data = Bytes::from(&[123u8; MAX_FRAME_SIZE + 1][..]);
-        let bad_msg = Elem::Data{ substream_id: 123, endpoint, data };
+        let bad_msg = Elem::Data {
+            substream_id: 123,
+            endpoint,
+            data,
+        };
```
Comment (twittner) [inline muxers/mplex/src/codec.rs]: Again, terrible.

File: muxers/mplex/src/lib.rs [hunk 4]
```diff
@@ -100,7 +99,7 @@ impl MplexConfig {
     #[inline]
     fn upgrade<C>(self, i: C, endpoint: Endpoint) -> Multiplex<C>
     where
-        C: AsyncRead + AsyncWrite
+        C: AsyncRead + AsyncWrite,
```
Comment (twittner) [inline muxers/mplex/src/lib.rs]: I do not get why rustfmt sometimes wants to add unnecessary commas and at other times it removes them.

**Comment by twittner** [comment]
Well, as far as I can remember we never discussed formatting in this project *until* the uglification of the code thanks to rustfmt, so rather than putting an issue to rest, it created one. It is beyond me why one would want to introduce a tool that decreases the readability of code just because. If we would have constant arguments about the placements of commas, braces and newlines I could see your point, but we do not.

**Comment by dvdplm** [comment]
As long as it controversial and irritates people we should probably desist. :)

**Comment by twittner** [comment]
@tomaka: What I meant is that we had no arguments about a particular formatting style.

**Pull request closed without merging**